gamification-evm-services/src/main/java/io/meeds/gamification/evm/blockchain/BlockchainConfigurationProperties.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ * Copyright (C) 2020 - 2024 Meeds Association contact@meeds.io
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package io.meeds.gamification.evm.blockchain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "gamification.evm.blockchain")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class BlockchainConfigurationProperties {
+
+  private String polygonNetworkUrl = "https://polygon-mumbai.g.alchemy.com/v2/5grQcqG3YrkpuxDbfhXE8HgPv04_iwSK";
```

Comment:
Never share this, it's similar to a password

```suggestion
  private String polygonNetworkUrl;
```